src/ProofOfConcepts/DeleteKafkaInstance/DeleteKafkaInstance.stories.tsx

```diff
@@ -0,0 +1,186 @@
+    target: "withError",
+  },
+  { target: "closed" },
+];
+const makeModalStoryMachine = ({
+  id,
+  initial = "open",
+  isAsync,
+  willFail,
+}: {
+  id: string;
+  initial?: string;
+  isAsync: boolean;
+  willFail: boolean;
+}) =>
+  createMachine({
+    initial,
+    id,
+    context: {
+      isAsync,
+      willFail,
+    },
+    states: {
+      closed: {
+        on: {
+          OPEN: "open",
+        },
+      },
+      open: {
+        on: {
+          DELETE: onDelete,
+        },
+      },
+      deleting: {
+        after: {
+          1000: [
+            { cond: (context) => context.willFail, target: "withError" },
+            { cond: () => initial !== "deleting", target: "closed" },
+          ],
+        },
+        on: {
+          CLOSE: "deleting",
+        },
+      },
+      withError: {
+        on: {
+          DELETE: onDelete,
+        },
+      },
+    },
+    on: {
+      CLOSE: "closed",
+    },
+  });
+
+const Template: ComponentStory<typeof DeleteModal> = (
+  { onDelete: onDeleteAction, onCancel: onCancelAction, children, ...args },
+  { id, parameters }
+) => {
+  const [state, send] = useMachine(
+    () =>
+      makeModalStoryMachine({
+        id,
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        initial: parameters.initialState,
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        isAsync: parameters.isDeleteAsync,
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        willFail: parameters.willDeleteFail,
+      }),
+    { devTools: true }
+  );
+
+  const onDelete = () => {
+    onDeleteAction && onDeleteAction();
+    send("DELETE");
+  };
+  const onCancel = () => {
+    onCancelAction && onCancelAction();
+    send("CLOSE");
+  };
+
+  return (
+    <div style={{ minHeight: 500, height: "100%" }}>
+      <DeleteModal
+        {...args}
+        isDeleting={state.value === "deleting"}
+        isModalOpen={state.value !== "closed"}
+        appendTo={() => {
+          return (
+            document.getElementById(`story--${id}`) ||
+            document.getElementById("root") ||
+            document.body
+          );
+        }}
+        onCancel={onCancel}
+        onDelete={onDelete}
+      >
+        {state.value === "withError" && (
+          <Alert variant="danger" title="Danger alert title" isInline>
+            <p>This should tell the user more information about the alert.</p>
+          </Alert>
+        )}
+        {children}
+      </DeleteModal>
+      <button onClick={() => send("OPEN")}>Open modal</button>
+    </div>
+  );
+};
+
+const fillConfirmation: PlayFunction<
+  ReactFramework,
+  DeleteModalProps
+> = async ({ canvasElement, args }) => {
+  const story = within(canvasElement);
+  const confirmationValue = args.confirmationValue || "resource name";
+  await userEvent.type(
+    await story.findByLabelText(`Type ${confirmationValue} to confirm`),
```

Comment:
Here I think that just the resource name or {confirmationValue} would be in bold and not the rest

Reply:
Yeah I will make those changes.